index.Rmd

```diff
@@ -19,7 +19,7 @@ This WDL workflow development guide shows a bioinformatics workflow developer ho
 
 ## Target Audience
 
-The course is intended for first time developers of the [WDL workflow language](https://github.com/openwdl/wdl), who wants to iteratively develop a WDL bioinformatics workflow. The audience should be able to run a WDL workflow on a computing engine of their choice, such as Cromwell, miniWDL, or a cloud computing environment such as Terra, AnVIL, or Dockstore. 
+The course is intended for first time developers of the [WDL workflow language](https://github.com/openwdl/wdl), who wants to iteratively develop a WDL bioinformatics workflow. The audience should be able to comprehend WDL syntax, and should be able to run a WDL workflow on a computing engine of their choice, such as Cromwell, miniWDL, or a cloud computing environment such as Terra, AnVIL, or Dockstore. 
```

Comment:
>  and should be able to run a WDL workflow on a computing engine of their choice, such as Cromwell, miniWDL, or a cloud computing environment such as Terra, AnVIL, or Dockstore. 

I think this is fair for what we're trying to do, but it does contradict what was said in the intro section in the previous PR. Should I reword that part of the introduction?

Reply:
Sorry, I should have clarified -- in part 1 of the tutorial, we mention that being able to run a WDL isn't a requirement for this course. The first time I read this passage here in part 2, I thought "and should be able to run a WDL workflow on a computing engine" contradicted that. But reading it again, it really isn't a contradiction.